script/verdaccio.js

```diff
@@ -0,0 +1,220 @@
+  process.chdir(sdkTestDir);
+  
+  // Clone SDK from github
+  logger.log(`Cloning ${SDK_APP_NAME} into ${sdkTestDir}`);
+  
+  //Clean up
+  spawnOrFail('rm', ['-rf', `${SDK_APP_NAME}`]);
+  spawnOrFail('git', [
+    'clone',
+    'git@github.com:aws/amazon-chime-sdk-js.git'
+  ]);
+  // navigate in to cloned SDK
+  process.chdir(path.join(`./${SDK_APP_NAME}`));
+  
+  // Update package.json version
+  let package = fs.readFileSync('package.json', 'utf-8');
+
+  const packageJson = JSON.parse(package);
+  const originalVersion = packageJson.version;
+  const versions = originalVersion.split('.').map(x => parseInt(x));
+  versions[2] += 1;
+  const newVersion = versions.join('.');
+
+  logger.log(`Changing version number from ${originalVersion} to ${newVersion}`);
+
+  // Update version to new version
+  let versionFile = fs.readFileSync('./src/versioning/Versioning.ts', 'utf-8');
+  fs.writeFileSync('./src/versioning/Versioning.ts', versionFile.replace('return VERSION.semverString;', `return '${newVersion};'`));
+  fs.writeFileSync('package.json', package.replace(originalVersion, newVersion));
+  package = fs.readFileSync('package.json', 'utf-8');
+
+  logger.log(`Temporarily removing prepublishOnly and postpublish`);
+  fs.writeFileSync('package.json', package.replace(`"postpublish": "script/postpublish",`, ""));
+  package = fs.readFileSync('package.json', 'utf-8');
+  fs.writeFileSync('package.json', package.replace(`"prepublishOnly": "script/publish",`, ""));
+  
+  logger.log('Installing dependencies...');
+  spawnOrFail('npm', ['install']);
+  logger.log('Building...');
+  spawnOrFail('npm', ['run', 'build']);
+
+  try {
+    // Cleanup package before publishing new
+    logger.log('Unpublishing any existing packages from verdaccio.');
+    spawnOrFail('npm', [`unpublish`, '--registry', `${verdaccioEndpoint}`, '--force']);
+  } catch {
+    logger.log('No existing npm module in verdaccio.');
+  }
+  logger.log('Publishing package to verdaccio.');
+  spawnOrFail('npm', [`publish`, '--registry', `${verdaccioEndpoint}`]);
+  
+  process.chdir('./demos/browser');
+
+  spawnOrFail('npm', [`uninstall ${SDK_APP_NAME}`]);
+
+  //  Install SDK components package from local repo.
+  logger.log(`Installing ${SDK_APP_NAME} from local verdaccio server...`);
+  spawnOrFail('npm', [`install ${SDK_APP_NAME} --registry ${verdaccioEndpoint}`]);
+
+  let meetingV2File = fs.readFileSync('app/meetingV2/meetingV2.ts', 'utf-8');
+  fs.writeFileSync('app/meetingV2/meetingV2.ts', meetingV2File.replace('../../../../src/index', 'amazon-chime-sdk-js'));
+
+  logger.log(`Installing and Building...`);
+  cp.execSync('cd ../.. && npm run build');
+  cp.execSync('npm install');
+  cp.execSync('npm install aws-sdk');
+  cp.execSync('npm run build');
+  const demoAppClientProcess = cp.exec('node server.js');
+  applicationProcesses.push(demoAppClientProcess);
+  
+  logger.log(`
+      Starting V1 meeting demo app. Please wait for the server to start up.
```

Comment:
This should be `V2` right? Also are we not including `V1` testing run?

Reply:
Yes, it should be V2, thanks